app/models/report.rb

```diff
@@ -9,24 +9,33 @@
 #  report_data    :binary
 #  created_at     :datetime         not null
 #  updated_at     :datetime         not null
+#  guid           :string
+#  properties     :jsonb
 #
 
-# Because run_and_update! needs to generate CSV
-require 'csv'
-
 class Report < ApplicationRecord
-  # Status names for ETL generated reports
-  COMPLETE = 'complete'.freeze
-  WAITING = 'waiting'.freeze
 
+  jsonb_accessor :properties,
+                 etl: :boolean,
```

Comment:
`etl` might need to change here as well if we decide to change it

Reply:
@ymao2 @willmcb 

Sorry, I'm now confused! Is this meant to mean background or warehouse? Should we change it?

Reply:
@Polsonby , it is warehouse thing,  there is another flag background_job is beneath it,  this flag is defined from report_type table, storing it in the report again is kind of duplication, let me remove it